internal/config/config.go

```diff
@@ -12,16 +12,17 @@ import (
 
 // Config global
 type Config struct {
-	URI      string     `json:"uri" yaml:"uri" env:"URI"`
-	Env      string     `json:"env" yaml:"env" env:"ENV"`             // The environment (eg: prd, stg)
-	AppName  string     `json:"appName" yaml:"appName" env:"APPNAME"` // app name used for monitoring
-	Domain   string     `json:"domain" yaml:"domain" env:"DOMAIN"`
-	Readers  Readers    `json:"readers" yaml:"readers" env:"READERS"`
-	Writers  Writers    `json:"writers" yaml:"writers" env:"WRITERS"`
-	Storage  Storage    `json:"storage" yaml:"storage" env:"STORAGE"`
-	Tables   Tables     `json:"tables" yaml:"tables" env:"TABLES"`
-	Statsd   *StatsD    `json:"statsd,omitempty" yaml:"statsd" env:"STATSD"`
-	Computed []Computed `json:"computed" yaml:"computed" env:"COMPUTED"`
+	URI       string     `json:"uri" yaml:"uri" env:"URI"`
+	SchemaURI string     `json:"schemaUri" yaml:"schemaUri" env:"SCHEMAURI"`
```

Comment:
Instead, can we make `Tables.Timeseries.Schema` a string and, similarly to scripts, accept an inline schema or a URI?